src/index.js

```diff
@@ -0,0 +1 @@
+"use strict";
```

Comment:
This file shouldn't be here. If you have correctly set up the `tsconfig.json` (as you have done) all the transpilled files should be outputted only in the `./build` directory. Probably this is a leftover but you must delete it.